kuma/static/styles/components/wiki/quick-links.scss

```diff
@@ -37,7 +37,7 @@
     }
 
     li li {
-        @include bidi(((padding-left, $grid-spacing, padding-right, 0),));
+        @include bidi(((padding-left, $grid-spacing, padding-right, 0)));
```

Comment:
Looks like there’s [an error in calling this](https://travis-ci.org/mozilla/kuma/jobs/442037276#L2606).

https://github.com/mozilla/kuma/blob/f088e94455be79acae6bb3a9e932c18e3338db4d/kuma/static/styles/includes/_mixins.scss#L378-L383

Based on the above, this needs the trailing comma which you removed 🤦‍🏻‍♂️

Reply:
Ooooh, bad rebase. Will fixup.

Reply:
Oh no, it was prettier that removed the `,`